README.md

```diff
@@ -60,25 +68,7 @@ export default Ember.Route.extend({
 })
 ```
 
-Once we have setup our attributes we can invoke them in the template.
-
-```hbs
-{{!-- app/templates/index.hbs --}}
-{{#impagination-dataset 
-  fetch=fetch 
-  page-size=pageSize 
-  load-horizon=loadHorizon 
-  unload-horizon=unloadHorizon 
-  read-offset=readOffset 
-  as |records|}}
-  <div class="records">Total Records: {{records.length}}</div>
-  {{#each records as |record|}}
-    <div class="record">Record {{record.id}}</div>
-  {{/each}}
-{{/impagination-dataset}}
-```
-
-Immediately, this will call fetch twice (for records 0-4, and 5-9)
+This setup will immediatly call fetch twice (for records 0-4 [page 0] and records 5-9 [page 1])
```

Comment:
> Because we're using a simple each, it will eagerly iterate through all the pages in the dataset which will result in first the initial pages, and finally the rest of the pages being loaded.  In most cases, using a simple `{{each}}` would defeat the purpose of using a data layer like `ember-impagination`, and for truly infinite datasets might result in truly infinite loops :scream: